src/shipit_dashboard/setup.py

```diff
@@ -33,4 +33,9 @@
     include_package_data=True,
     zip_safe=False,
     license='MPL2',
+    entry_points={
+        'console_scripts': [
+            'shipit-bot= shipit_bot.cli:main',
```

Comment:
can a command line be called ``shipit-dashboard-bot``. A reason for this is that this bot is very shipit_dashboard specific and names should reflect that.

Also, why is code placed outside  ``src/shipit_dashboard/shipit_dashboard/`` folder? I would expect even the CLI code be inside there ... even if not used. I would just still all the command line specific folder in ``cli.py`` module